### PR TITLE
[15.0][FIX] base_duplicate_security_group: use a proper hook

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa TT3142 TT42816

ping @sergio-teruel @CarlosRoca13 @pilarvargas-tecnativa

### CODE REVIEW

File: base_duplicate_security_group/static/src/js/duplicate.esm.js [hunk 1]
```diff
@@ -11,11 +11,14 @@ import session from "web.session";
...
-            "base_duplicate_security_group.group_duplicate_records"
-        );
-        return this._super(...arguments);
+    init() {
+        this._super(...arguments);
+        const base_group = "base_duplicate_security_group.group_duplicate_records";
+        session.user_has_group(base_group).then((result) => {
+            if (!result) {
+                this.controllerParams.activeActions.duplicate = false;
+            }
```
Comment (CarlosRoca13) [inline base_duplicate_security_group/static/src/js/duplicate.esm.js:20]: Is `this.controllerParams.activeActions.duplicate` activated without defining it as true?

Comment (chienandalu) [reply base_duplicate_security_group/static/src/js/duplicate.esm.js:20]: Yup -> https://github.com/odoo/odoo/blob/1785047d58f28ba11e11e40ba134e89bb235fa8b/addons/web/static/src/legacy/js/views/abstract_view.js#L146

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by chienandalu** [comment]
please check @sergio-teruel

**Comment by pilarvargas-tecnativa** [comment]
The tests have been corrected to fix the same problem that occurred with the base_import_security_group module.
PR: https://github.com/OCA/server-ux/pull/665

**Review by stefan-tecnativa — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**